README.md

```diff
@@ -332,6 +343,7 @@ if (args.help) {
 #### Is `--foo` the same as `--foo=true`?
   * No.
   * `--foo` will parse to `{ args: { "foo": [""] } }` & `--foo=true` to ` { args: { "foo": ["true"] } }` respectively
+🙏
```

Comment:
I hope this emoji means this is a good thing? Not coercing/inferring types is a big part of this approach.